packages/shared/components/Select/DarkStyledSelect.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 Gravitational, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import styled from 'styled-components';
+import { StyledSelect } from './Select';
+
+const StyledDarkSelect = styled(StyledSelect)(
+  ({ theme }) => `
+  .react-select-container {
+    background: transparent;
+  }
+  
+  .react-select__control,
+  .react-select__control--is-focused {
+    border-color: #FFF;
+    min-height: 34px;
+  }
+  
+  .react-select__value-container {
+    padding: 0 8px;
+  }
+
+  .react-select__single-value {
+    color: ${theme.colors.text.primary}
+  }
+
+  .react-select__indicator,
+  .react-select__dropdown-indicator {
+    padding: 4px 8px;
+    color: #666;
+    &:hover {
+      color: #999 !important;
+    }
+  }
+
+  .react-select__control {
+    border-color: rgba(255, 255, 255, 0.24);
+    color: ${theme.colors.text.secondary};
+
+    &:focus, &:active {
+      background-color: ${theme.colors.primary.lighter};
+    }
+
+    &:hover {
+      border-color: rgba(255, 255, 255, 0.24);
+      background-color: ${theme.colors.primary.lighter};
+      .react-select__dropdown-indicator {
+        color: #666;
+      }
+    }
+  }
+
+  .react-select__control--is-focused {
```

Comment:
```suggestion
  .react-select__control--menu-is-open {
```
I think this is better, it only highlights when its opened, and removes focus afterwards

Reply:
this will do the trick